x-pack/plugins/canvas/public/components/workpad_interactive_page/index.js

```diff
@@ -0,0 +1,170 @@
+/*
```

Comment:
Q: why not just keep this in `public/components/workpad_page/`? It's not like this is really reusable, and it's not meant to be used on its own anyway (you use it via `WorkpadPage`).

Reply:
To extend on my main-thread reply, I wasn't aware of the convention that only reusable components should be in the top level, as many (or most?) of the other components also have a use only in another component. 

While I listed reusability elements for `StaticPage` (presentation mode, kiosk, page manager, static mobile, bulk export) I don't currently see much reusability for the interactive page, at least in Canvas (eventually the Kibana Dashboard may become something like the interactive page, except with tiling window constraint guides rather than freeform, at least initially). 

Not sure if an asymmetric solution (ie. put only the `InteractivePage` under `components/workpad_page/`) would be nicer, and if so, could it retain its own directory so all the files don't intermingle. I don't really mind either way, just need to grok what the component structuring convention is.

Maybe we could tweak the guidelines and make a follow-up PR just for the moving.

Reply:
...thanks, just saw your main thread reply, makes sense!

Reply:
Sadly, there hasn't been much convention, so `public/components` has a ton of one-off things that are really just child components that are not reused. This isn't a new problem or anything.

Reply:
Thanks, I'll simply move these two directories under `components/workpad_page/` so they're together now, but easily separable.